tests/map/options.test.ts

```diff
@@ -61,4 +64,63 @@ describe('MapOptions', () => {
             assert(max.innerHTML === 'max:');
         }
     });
+
+    it('selected property changes when changed', () => {
+        const root = document.createElement('div');
+        const options = new MapOptions(root, DUMMY_PROPERTIES, DUMMY_CALLBACK);
+
+        checkPropertySelect(options.x, 'x');
+        checkPropertySelect(options.y, 'y');
+        checkPropertySelect(options.z, 'z');
+        checkPropertySelect(options.color, 'color');
+
+        function checkPropertySelect(axisOptions: AxisOptions, axisName: string) {
+            const selectElement = getByID<HTMLSelectElement>(`chsp-${axisName}`);
+            selectElement.value = 'second';
+            selectElement.dispatchEvent(new window.Event('change'));
+            assert(axisOptions.property.value === 'second');
+        }
+    });
+
+    it('selected range changes when changed', () => {
+        const root = document.createElement('div');
+        const options = new MapOptions(root, DUMMY_PROPERTIES, DUMMY_CALLBACK);
+
+        checkRangeSelect(options.x, 'x');
+        checkRangeSelect(options.y, 'y');
+        checkRangeSelect(options.z, 'z');
```

Comment:
why not the color here?

Reply:
Still an open question =)

Reply:
sorry, i didn't notice it earlier. i just added a new commit